templates/available_hosts.html

```diff
@@ -13,37 +13,39 @@
         <div class="col">
             <div class="table-responsive-sm">
                 <br>
-                <form hx-post="/hosts/lease" , hx-target="body">
+                <form id="host-form" hx-post="/hosts/lease" hx-target="body">
                     <fieldset>
                         <legend>Available hosts</legend>
                         <fieldset>
                             <legend>Select lease period</legend>
                             <label for="days">Days:</label>
-                            <input type="number" id="days" name="days" min="0" max="14" value="0">
+                            <input type="number" id="days" name="days" min="0" max="62" value="0">
                             <label for="days">Hours:</label>
-                            <input type="number" id="hours" name="hours" min="0" max="24" value="1">
+                            <input type="number" id="hours" name="hours" min="0" max="23" value="1">
                         </fieldset>
+                        <button type="submit">Lease selected</button>
+                        <button hx-validate="true" hx-post="/hosts/lease/random">Lease random</button>
```

Comment:
This does not trigger form validation as the above button but at least it does not submit the form. I guess this is the best I can do without doing any JS and custom validation logic

Reply:
Ok
backend validation we have too